vocabularies-gsq/retention-status.ttl

```diff
@@ -3,115 +3,142 @@ PREFIX agldwgstatus: <https://linked.data.gov.au/def/reg-statuses/>
 PREFIX reg: <http://purl.org/linked-data/registry#>
+PREFIX retstat: <http://linked.data.gov.au/def/retention-status/>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX sdo: <https://schema.org/>
 PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
 PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 
-:gsq-hold
+retstat:gsq-hold
     a skos:Concept ;
     rdfs:isDefinedBy cs: ;
-    skos:definition "To be held until the requesting officer or their director relinquishes the hold."@en ;
+    skos:definition "To be held until the requesting officer or their director relinquishes the hold. Can be viewed and sampled, but not disposed."@en ;
     skos:historyNote "Created from the WIMS MIRO brainstorming session" ;
     skos:inScheme cs: ;
     skos:notation "gsq-hold" ;
     skos:prefLabel "GSQ Hold"@en ;
     skos:topConceptOf cs: ;
 .
 
-:type-section
+retstat:type-section
     a skos:Concept ;
     rdfs:isDefinedBy cs: ;
-    skos:definition "To be held indefinitely or until replaced by a sample which meets the strict criteria set for type sections. Type sections are the part of the definition of a lithostratigraphic unit, without a type section the unit is considered invalid and has no official standing. Refer to https://www.ga.gov.au/data-pubs/datastandards/stratigraphic-units/stratigraphic-type-sections"@en ;
+    skos:definition "To be held indefinitely or until replaced by a sample which meets the strict criteria set for type sections. Type sections are the part of the definition of a lithostratigraphic unit, without a type section the unit is considered invalid and has no official standing. Can be viewed and sampled (With approval), but not disposed. Refer to https://www.ga.gov.au/data-pubs/datastandards/stratigraphic-units/stratigraphic-type-sections"@en ;
     skos:historyNote "Created from the WIMS MIRO brainstorming session" ;
     skos:inScheme cs: ;
     skos:notation "type-section" ;
-    skos:alias "reference-section" ;
+    skos:alias "type-section" ;
     skos:prefLabel "Type Section"@en ;
     skos:topConceptOf cs: ;
 .
 
-:university-hold
+retstat:reference-collection
     a skos:Concept ;
     rdfs:isDefinedBy cs: ;
-    skos:definition "Held by the Gelogical Survey of Queensland on behalf of a university. Not to be disposed of until the requesting acadameic relinquishes the hold, or the hold is released by a GSQ Director."@en ;
+    skos:broader retstat:type-section ;
+    skos:definition "To be held indefinitely or until replaced by a sample which meets the strict criteria set.  Can be viewed and sampled (With approval), but not disposed."@en ;
     skos:historyNote "Created from the WIMS MIRO brainstorming session" ;
     skos:inScheme cs: ;
-    skos:notation "approved-access-only" ;
-    skos:prefLabel "Approved Access Only"@en ;
+    skos:notation "reference-collection" ;
+    skos:alias "reference-collection" ;
+    skos:prefLabel "Reference Collection"@en ;
     skos:topConceptOf cs: ;
 .
 
-:proposed-disposal

```

Comment:
I thought Proposed for disposal is an important status for WIMS. Has it been removed?